sycl/doc/extensions/SPIRV/SPV_INTEL_usm_storage_classes.asciidoc

```diff
@@ -96,30 +100,38 @@ Modify Section 3.31, Capability, adding a row to the Capability table:
 
 === Instructions
 
-Modify Section 3.36.11, Conversion Instructions, replacing the descriptions of *OpPtrCastToGeneric* and *OpGenericCastToPtr* with the following.  Their required capabilities and operands remain unchanged.  
+Modify Section 3.36.11, Conversion Instructions, adding two new instructions as follows:
 
 |===
-*OpPtrCastToGeneric*
+3+^| *OpPtrCastToCrossWorkgroupINTEL*
 
-Converts a pointer's Storage Class to a more generic class.
+Converts a pointer's Storage Class from a more specific class to *CrossWorkgroup*.
```

Comment:
```suggestion
Converts a pointer's Storage Class from a more specific class to *CrossWorkgroup* or *Generic*.
```

Reply:
I discussed this with @bashbaug and he prefers not overlapping with the functionality of the existing conversion instructions by restricting the new conversions in this way.

Reply:
Thus, if you want to convert from device or host to generic you'd need to use two instructions: one to convert to CrossWorkgroup and then a second to convert to Generic.

Reply:
Ok